depends/windows/nasmw/nasmw.txt

```diff
@@ -1 +1 @@
-nasmw http://mirrors.xbmc.org/build-deps/sources/nasm-2.11.06.zip
+nasmw http://www.nasm.us/pub/nasm/releasebuilds/2.14.02/win32/nasm-2.14.02-win32.zip
```

Comment:
build from source

Reply:
It seems to need Perl which is not available. Could try to add perl.

Reply:
With a patch on depends becomes the Perl usage prevented, also is it now added to UWP, there it has also worked with compiled Version.